x-pack/plugins/enterprise_search/public/applications/app_search/components/layout/nav.test.tsx

```diff
@@ -8,7 +8,7 @@
 import { setMockValues } from '../../../__mocks__/kea_logic';
 
 jest.mock('../../../shared/layout', () => ({
-  generateNavLink: jest.fn(({ to }) => ({ href: to })),
+  generateNavLink: jest.fn(({ to, items }) => ({ href: to, items })),
```

Comment:
I'm thinking about DRYing out an importable mock for `shared/layout/generateNavLink` at some point since a few different nav files use it, but might save that for the final page template PR